Gemfile.lock

```diff
@@ -565,10 +565,9 @@ GEM
       rake (>= 0.8.7)
       rdoc (~> 3.4)
       thor (>= 0.14.6, < 2.0)
-    rainbow (2.2.2)
-      rake
+    rainbow (3.0.0)
     raindrops (0.13.0)
-    rake (10.5.0)
+    rake (12.3.1)
```

Comment:
watch out. AFAIK this version of Rake won't work with our version of RSpec Rails. It may cause an error with `#last_comment`.

Reply:
Ah thanks. I now discovered the `--conservative` flag of `bundle update`. That's much better for our needs. On the other hand, if we know that that version doesn't work, we should blacklist it in the Gemfile, shouldn't we?

See my additional commits. Seems all fine now.